.browserslistrc

```diff
@@ -0,0 +1 @@
+defaults
```

Comment:
Hey overall it looks great Conor.  Your interface is clean.  Creating/deleting posts works and everything looks good.  I guess if you want to continue on this project, you would enable liking/commenting on posts.  Great work!